docs/modules/plugins/pages/plugin-db.adoc

```diff
@@ -1,19 +1,127 @@
 = Relational Database Plugin
 
 The plugin provides capabilities to work with relational databases via
-https://en.wikipedia.org/wiki/Java_Database_Connectivity[JDBC] and contains
-a set of the steps for comparison of data tables.
+JDBC and contains a set of the steps for comparison of data tables.
+
+Java Database Connectivity (https://en.wikipedia.org/wiki/Java_Database_Connectivity[JDBC]) is an application programming interface for Java, which defines how a client may access a database.
+
+Every JDBC requires using specific driver. JDBC drivers are client-side adapters (installed on the client machine, not on the server) that convert requests from Java programs to a protocol that the database management system can understand.
 
 == Installation
 
+. Add plugin dependency into build.gradle file:
++
 .build.gradle
 [source,gradle,subs="attributes+"]
 ----
 implementation(group: 'org.vividus', name: 'vividus-plugin-db', version: '{current-version}')
 ----
 
+. Define your JDBC driver and add appropriate dependency. Let's take PostgreSQL as an example:
++
+- Open JDBC page on https://mvnrepository.com/open-source/jdbc-drivers[maven repository];
+- Find driver for your database and open details (https://mvnrepository.com/artifact/org.postgresql/postgresql[PostgreSQL JDBC Driver]);
+- Choose and open version (https://mvnrepository.com/artifact/org.postgresql/postgresql/42.4.0[42.4.0]);
+- Go to the Gradle tab and copy implementation string;
+- Paste chosen dependency into your build.gradle file:
++
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.postgresql', name: 'postgresql', version: '42.4.0')
+----
++
+NOTE: In case Maven repository does not contain driver you need, please refer to the documentation of your database to define suitable JDBC driver.
+
 == Configuration
 
+=== Database properties
+
+For database interaction, you will need to set related properties. They have `db.connection.` prefix followed by the database name (`vividus` in the example). Come up with your own database name to use within the tests. Put properties into the `environment.properties` file:
+
+.environment.properties
+[source,properties,subs="attributes+"]
+----
+db.connection.vividus.url=jdbc:dbms://databaseUrl:port/databaseName?parameter1=true&parameter2=value <1>
+db.connection.vividus.username= <2>
+db.connection.vividus.password= <2>
+db.connection.vividus.driver-class-name=<3>
+----
+<1> Database address. Consists of:
+    - jdbc: - reference to JDBC;
+    - dbms: - Database Management System, e.g. `postgresql`;
+    - //databaseUrl - address to your database;
+    - :port - port number, e.g. 5432;
+    - /databaseName - name of your database;
+    - parameter - any additional parameters you need to pass;
++
+IMPORTANT: URL structure slightly changes depending on the database management system. Below you can find templates for different DBMS
++
+.Click to see URL templates
```

Comment:
let's move these templates to `Setup examples` so to keep information about particular DBMS-s grouped (also you can add a reference to `Setup examples` from this point)